index.html

```diff
@@ -47,7 +72,27 @@
 
 	</div>
 
```

Comment:
I think you should add a `<br>` or two here to ensure that the footer doesn't overlap the contributor images on mobile.